browser/net/brave_block_safebrowsing_urls.cc

```diff
@@ -20,6 +20,9 @@ bool IsSafeBrowsingReportingURL(const GURL& gurl) {
       URLPattern(
           URLPattern::SCHEME_HTTPS,
           "https://sb-ssl.google.com/safebrowsing/clientreport/download*"),
+      URLPattern(
+          URLPattern::SCHEME_HTTPS,
+          "https://sb-ssl.google.com/safebrowsing/clientreport/crx-list-info"),
```

Comment:
May be add a test in `brave_static_redirect_network_delegate_helper_unittest.cc`

Reply:
Added in the latest revision. Turns out I got the URL wrong. Good thing you asked for a test!